local-setup/launch.py

```diff
@@ -196,7 +196,7 @@ def get_flags(index, worker):
 
     return list(filter(None, [
         "--clean-reset",
-        "-T", "wss://localhost",
+        "-T", "ws://localhost",
```

Comment:
could this impact production deployments?

Reply:
nope, it related to not more than just local run. This particular change allows to run ` ./litentry-cli print-sgx-metadata-raw`, which we need for running integration tests locally. Otherwise the execution will hang and won't go further

Reply:
It hangs because it tries to use default value

```
	#[clap(short = 'U', long, default_value_t = String::from("ws://127.0.0.1"))]
	worker_url: String,
```

You could explicitly pass `-U wss://localhost` to CLI but having it working out of the box is more handy.

Reply:
Thanks for the tip, modified package.json instead of python lanucher `./bin/litentry-cli -U wss://localhost print-sgx-metadata-raw`